includes/object-cache.php

```diff
@@ -679,7 +683,14 @@ public function flush($delay = 0)
                     $this->redis instanceof Predis\Client ? 0 : []
                 ));
             } else {
-                $result = $this->parse_redis_response($this->redis->flushdb());
+                if (defined('WP_REDIS_CLUSTER') && defined('WP_REDIS_CLIENT') && strcasecmp('pecl', WP_REDIS_CLIENT) === 0) {
+                    foreach(WP_REDIS_CLUSTER as $host) {
+                        $parsed_host = parse_url($host);
+                        $result = $this->parse_redis_response($this->redis->flushdb($parsed_host['host']));
+                    }
```

Comment:
I never use Redis Cluster. Is it necessary to call `FLUSHDB` on all instances, wouldn't one be enough?

Reply:
Correct. The documentation for PHPRedis claims that FLUSHDB is a single-node interaction, and must be called on a specific node: https://github.com/phpredis/phpredis/blob/develop/cluster.markdown#directed-node-commands.

Based on the example code there, though, I may try seeing if I can use the suggested syntax:

```
foreach($this-redis->_masters() as $master) {
    $this->redis->flushdb($master);
}
```

Will report back if that seems to work, and update my PR accordingly. This is all new to me, too, so apologies for all the different versions trying to get something to work.